lib/questions.rb

```diff
@@ -213,14 +252,22 @@ def count_words_of_each_length_in_a_file(file_path)
 # implement fizzbuzz without modulo, i.e. the % method
 # go from 1 to 100
 # (there's no RSpec test for this one)
-def fizzbuzz_without_modulo
+def fizzbuzz_without_modulo(number, div)
 end
 
 # print the lyrics of the song 99 bottles of beer on the wall
 # http://www.99-bottles-of-beer.net/lyrics.html
-# make sure you use the singular when you have one bottle of 
+# make sure you use the singular when you have one bottle of
 # beer on the wall, and print 'no more bottles of beer on the wall'
 # at the end.
 # (there's no RSpec test for this one)
 def ninety_nine_bottles_of_beer
+  bottles_of_beer = 99
+
+  while bottles_of_beer >= 0
+    puts "#{bottles_of_beer} bottles of beer on the wall." if bottles_of_beer > 1
+    puts "#{bottles_of_beer} bottle of beer on the wall." if bottles_of_beer == 1
```

Comment:
Line is too long. [81/80]<br>Do not write to stdout. Use Rails' logger if you want to log.